problem_builder/answer.py

```diff
@@ -64,6 +68,40 @@ def _get_student_id(self):
         except AttributeError:
             return self.scope_ids.user_id
 
+    @staticmethod
+    def _fetch_model_object(name, student_id, course_id):
+        sql_query = '''SELECT * FROM problem_builder_answer
+                         WHERE name = %s
+                           AND student_id = %s
+                           AND (course_key = %s
+                                OR course_id = %s)'''
```

Comment:
@maxrothman Only because I was not familiar with django `Q` objects and didn't know django ORM supports `OR` queries. Thanks for the hint, I opened #136 to convert the raw SQL query to `Q` objects - can you please review?

If it looks good, I'll update the version number at https://github.com/edx/edx-platform/pull/14044 and we can finally get that one merged.